packages/cli/commands/import/workflow.ts

```diff
@@ -111,6 +121,10 @@ export class ImportWorkflowsCommand extends Command {
 							});
 						}
 
+						if (Object.prototype.hasOwnProperty.call(workflow, 'tags')) {
+							await setTagsForImport(transactionManager, workflow, tags);
+						}
```

Comment:
1. I would simplify the check with `in` and be more defensive and extract it all into a helper: `const hasTags = (workflow) => 'tags' in workflow && Array.isArray(workflow.tags) && workflow.tags.length > 0`. Same for the other call. This also slims down `setTagsForImport`, if you agree that validation should not be its responsibility.
2. Subjective, but since the definition of `setTagsForImport` is only needed for this import command, I'd relocate it here instead of `TagHelpers.ts` along with `findOrCreateTag` and `createTag`.

Reply:
Added helper. Will keep here. It's a tags helper.. it fits.